bin/dotd

```diff
@@ -110,6 +113,19 @@ def open_pull_request(base, compare, title)
   open_url "https://github.com/code-dot-org/code-dot-org/compare/#{base}...#{compare}?expand=1&title=#{CGI.escape title}"
 end
 
+# Prints the commit messages of commits present in branch head but not branch
+# base.
+# @param base [String] The base branch to compare.
+# @param head [String] The head branch to compare against.
+# @example For a DTT: view_commits(base: 'test', head: 'staging')
+def view_commits(base:, head:)
+  GitHub.compare(base: base, head: head).each do |commit_message|
+    # Empirically, commit messages often contain blank lines. For console
+    # readability, we attempt to remove them.
+    puts commit_message.gsub("\n\n", "\n")
```

Comment:
I wonder if it'd be nicer to just print the first line of each commit.  Commit messages conventionally have a 'topic' line, a blank line, and more content to be friendly to summary output like this:
```
* dc98101 - (HEAD -> minitest-junit-reporting, origin/minitest-junit-reporting) Add JUnit reports for cucumber [test all] (16 hours ago) <Brad Buchanan>
* 029f1da - Move apps JUnit report to apps/ directory (16 hours ago) <Brad Buchanan>
* 3ca2e8b - Add JUnit reports for shared tests on Circle CI [test all] (19 hours ago) <Brad Buchanan>
* cf4b04f - Provide JUnit reports for dashboard tests on Circle [test all] (19 hours ago) <Brad Buchanan>
*   5d84067 - Merge pull request #13630 from code-dot-org/section-projects-sorting (2 days ago) <David Bailey>
```
I know a lot of my commits are one-line messages but for a handful I practically write the whole PR description - I wouldn't want someone to have to deal with that here.